modular_pariah/modules/orderlies/code/orderly.dm

```diff
@@ -0,0 +1,46 @@
+/datum/job/orderly
+	title = JOB_ORDERLY
+	alt_titles = list("Orderly", "Medical Guard")
+	description = "Defend the medical department, hold down idiots who refuse the vaccine, flex your muscles at people who fuck with medical."
```

Comment:
I feel like this job description would promote a pretty aggressive playstyle, sort of a medbay-based sec officer rather than a guard, but I don't know what (if anything) to replace it with.